docs/installation.md

```diff
@@ -36,11 +36,34 @@ Now in the `~/govuk` directory, run the following commands.
 git clone git@github.com:alphagov/govuk-docker.git
 cd govuk-docker
 bundle install
-govuk-docker setup
+bundle exec bin/setup
 ```
 
 You can now [clone and setup the apps you need](../README.md#Usage), after which you can do things like run tests and startup the app in your browser. If this doesn't work for whatever reason, follow the [instructions on how to resolve setup issues](#how-tos).
 
+### Shortcuts
+
+Typing the [full commands](../README.md#usage) is likely to get tiring. We recommend adding the following functions (and aliases) to your shell config, which figure out the application you're running based on the name of the current directory, matching it with the corresponding docker-compose.yml in govuk-docker/services.
+
+```
+function govuk_docker_run () {
+  STACK=lite
+  govuk-docker run $(basename $(pwd))-$STACK "$@"
+}
+
+function govuk_docker_up () {
+  STACK=${1:-app}
+  govuk-docker up $(basename $(pwd))-$STACK
+}
+
+alias gd="govuk-docker"
+alias gdr="govuk_docker_run"
+alias gdu="govuk_docker_up"
+alias gdbe="govuk_docker_run bundle exec"
+```
```

Comment:
I'm pleased that only gd conflicts with the [oh my zsh git aliases](https://github.com/robbyrussell/oh-my-zsh/wiki/Cheatsheet#git)